health/notifications/alarm-notify.sh.in

```diff
@@ -2638,117 +2683,747 @@ EOF
+                    <table border=\"0\" cellpadding=\"0\" cellspacing=\"0\" role=\"presentation\" width=\"100%\">
+                      <tbody>
+                      <tr>
+                        <td style=\"vertical-align:top;padding-top:13px;\">
+                          <table border=\"0\" cellpadding=\"0\" cellspacing=\"0\" role=\"presentation\" style width=\"100%\">
+                            <tbody>
+                            <tr>
+                              <td align=\"right\" style=\"font-size:0px;padding:10px 25px;word-break:break-word;\">
+                                <div style=\"font-family:Open Sans, sans-serif;font-size:13px;line-height:1;text-align:right;color:#555555;\"><span style=\"background-color:#FFF8E1; border: 1px solid #FFC300; border-radius:36px; padding: 2px 12px; margin-top: 20px; white-space: nowrap\">
+              Warning for ${elapsed_txt}
+           </span></div>
+                              </td>
+                            </tr>
+                            </tbody>
+                          </table>
+                        </td>
+                      </tr>
+                      </tbody>
+                    </table>
+                  </div>
+                  <!--[if mso | IE]></td></tr></table><![endif]-->
+                </td>
+              </tr>
+              </tbody>
+            </table>
+          </div>
+       "
+
+   done <<<"$total_warn_alarms,"
+fi
+
+if [ -n "$total_crit_alarms" ]; then
+   while read -d, -r pair; do
+       IFS='=' read -r key val <<<"$pair"
+
+       date_c=$(date --date=@${val} "${date_format}" 2>/dev/null)
+       [ -z "${date_c}" ] && date_c=$(date "${date_format}" 2>/dev/null)
+       [ -z "${date_c}" ] && date_c=$(date --date=@${val} 2>/dev/null)
+       [ -z "${date_c}" ] && date_c=$(date 2>/dev/null)
+
+       elapsed=$((now - val))
+
+       duration4human ${elapsed} >/dev/null
+       elapsed_txt="${REPLY}"
+
+       CRIT_ALARMS+="
+       <div class=\"set-font\" style=\"font-family: 'IBM Plex Sans', sans-serif; background: #FFFFFF; background-color: #FFFFFF; margin: 0px auto; max-width: 600px;\">
+            <table align=\"center\" border=\"0\" cellpadding=\"0\" cellspacing=\"0\" role=\"presentation\" style=\"background:#FFFFFF;background-color:#FFFFFF;width:100%;\">
+              <tbody>
+              <tr>
+                <td style=\"border-top:8px solid #F7F8F8;direction:ltr;font-size:0px;padding:20px 0;text-align:center;\">
+                  <!--[if mso | IE]><table role=\"presentation\" border=\"0\" cellpadding=\"0\" cellspacing=\"0\"><tr><td class=\"\" style=\"vertical-align:top;width:300px;\" ><![endif]-->
+                  <div
+                    class=\"mj-column-per-50 mj-outlook-group-fix\" style=\"font-size:0px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:50%;\"
+                  >
```

Comment:
This `>`  does not need to be in  a different line, please, fix this.

Reply:
Another thing, for `WARN_ALARMS`, the `<div class` was in the same line, and for some reason you split them here for `CRIT_ALARMS`, I could not understand, this was only typo?

Reply:
Hi Thiago! Yes, a typo.... will fix the formatting issues, many thanks!

Reply:
@thiagoftsm the reason why this is formatted this way is that this template is actually autogenerated code. The source for this is [here](https://github.com/netdata/dashboard/tree/master/src/email-templates/alert-notification). When we'll do any changes, we'll recompile it and paste it into alarm-notify.sh.in template. That's why there's no reason to make any formatting changes, because

1) we'll have to repeat them every time anyway
2) it will obscure the diff when you'll add new changes

Reply:
Thank you @jacekkolasa to explain. I imagined that this was typo, because this is not a pattern present for all tags.